parking_permits/services/traficom.py

```diff
@@ -1,12 +1,144 @@
-def get_vehicle_information(registration_number):
-    return {
-        "type": "Bensin",
-        "category": "M1",
-        "manufacturer": "Tesla",
-        "model": "Supreme",
-        "production_year": 2000,
-        "euro_class": 6,
-        "emission": 120,
-        "emission_type": "WLTP",
-        "last_inspection_date": "2022-08-13",
-    }
+import logging
+import xml.etree.ElementTree as ET
```

Comment:
Not sure if it worths the efforts now. Using xml ElementTree to deal with SOAP API might be cumbersome. There's a nice library that makes it easier to deal with SOAP APIs: https://docs.python-zeep.org/en/master/